stdlib/cmake/modules/StdlibOptions.cmake

```diff
@@ -169,8 +169,11 @@ option(SWIFT_STDLIB_HAS_COMMANDLINE
        TRUE)
 
 option(SWIFT_ENABLE_REFLECTION
-  "Build stdlib with support for runtime reflection and mirrors."
-  TRUE)
+       "Build stdlib with support for runtime reflection and mirrors."
+       TRUE)
+
+set(SWIFT_STDLIB_REFLECTION_METADATA "enabled" CACHE STRING
```

Comment:
Does this really want to default to `disabled` when `SWIFT_ENABLE_REFLECTION` is `FALSE`?

Reply:
I don't think I have a very strong position either way, but my reasoning to keep it default to "enabled" is that I view the  two "configurable behaviors" as completely separate things, and AFAICT the reflection metadata, despite the name, is used for more things than just to support the the reflection APIs the stdlib offers.

But again, I don't think I have a strong preference here, so if you still think it would make more sense change the default (under SWIFT_ENABLE_REFLECTION = FALSE), happy to do so.

Reply:
I'm happy with it the way it is, just wanted to be sure we'd thought about it.